Xamarin.Forms.Core/Cells/CellExtensions.cs

```diff
@@ -0,0 +1,39 @@
+using System;
+using Xamarin.Forms.Internals;
+
+namespace Xamarin.Forms
```

Comment:
Did we consider adding these extensions to `Xamarin.Forms.Internal`? Jason wanted to hide the coupling of the forms elements to the renderers from intellisense; these extensions will appear in the cells intellisesne.